builtin/providers/circonus/resource_circonus_check_httptrap_test.go

```diff
@@ -86,7 +86,7 @@ resource "circonus_check" "consul" {
   period = "60s"
 
   collector {
-    id = "/broker/1286"
+    id = "/broker/2110"
```

Comment:
I see, thanks for the explanation, it is - as I said - unrelated to this PR, but I think it would be worthwhile turning this into ENV variable at some point, so that the tests can run with any account.